cluster/gce/addons/konnectivity-agent/daemonset.yaml

```diff
@@ -47,13 +42,6 @@ spec:
                 fieldRef:
                   fieldPath: metadata.namespace
           resources:
-            limits:
+            requests:
```

Comment:
Do we need merge this change?

Reply:
I didn't notice any differences to the test passing rate suggesting that there's a bigger problem we need to tackle first.